src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -2598,52 +2355,47 @@ namespace WaterToAirHeatPumpSimple {
-                PartLoadRatio * state.dataWaterToAirHeatPumpSimple->LoadSideOutletHumRat +
-                (1.0 - PartLoadRatio) * state.dataWaterToAirHeatPumpSimple->LoadSideInletHumRat;
-            state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).OutletAirDBTemp =
-                PsyTdbFnHW(state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).OutletAirEnthalpy,
-                           state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).OutletAirHumRat);
-            state.dataWaterToAirHeatPumpSimple->PLRCorrLoadSideMdot = state.dataWaterToAirHeatPumpSimple->LoadSideMassFlowRate;
+            simpleWatertoAirHP.OutletAirEnthalpy =
+                PartLoadRatio * LoadSideFullOutletEnthalpy + (1.0 - PartLoadRatio) * state.dataWaterToAirHeatPumpSimple->LoadSideInletEnth;
+            simpleWatertoAirHP.OutletAirHumRat = PartLoadRatio * state.dataWaterToAirHeatPumpSimple->LoadSideOutletHumRat +
+                                                 (1.0 - PartLoadRatio) * state.dataWaterToAirHeatPumpSimple->LoadSideInletHumRat;
+            simpleWatertoAirHP.OutletAirDBTemp = Psychrometrics::PsyTdbFnHW(simpleWatertoAirHP.OutletAirEnthalpy, simpleWatertoAirHP.OutletAirHumRat);
         } else {
             // default to cycling fan, cycling compressor
-            state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).OutletAirEnthalpy = state.dataWaterToAirHeatPumpSimple->LoadSideOutletEnth;
-            state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).OutletAirHumRat = state.dataWaterToAirHeatPumpSimple->LoadSideOutletHumRat;
-            state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).OutletAirDBTemp = state.dataWaterToAirHeatPumpSimple->LoadSideOutletDBTemp;
-            state.dataWaterToAirHeatPumpSimple->PLRCorrLoadSideMdot = state.dataWaterToAirHeatPumpSimple->LoadSideMassFlowRate * PartLoadRatio;
+            simpleWatertoAirHP.OutletAirEnthalpy = LoadSideFullOutletEnthalpy;
+            simpleWatertoAirHP.OutletAirHumRat = state.dataWaterToAirHeatPumpSimple->LoadSideOutletHumRat;
+            simpleWatertoAirHP.OutletAirDBTemp = state.dataWaterToAirHeatPumpSimple->LoadSideOutletDBTemp;
         }
 
         // scale heat transfer rates to PLR and power to RTF
-        state.dataWaterToAirHeatPumpSimple->QLoadTotal *= PartLoadRatio;
-        state.dataWaterToAirHeatPumpSimple->QSensible *= PartLoadRatio;
+        simpleWatertoAirHP.QLoadTotal *= PartLoadRatio;
+        simpleWatertoAirHP.QLoadTotalReport = simpleWatertoAirHP.AirMassFlowRate *
+                                              (state.dataWaterToAirHeatPumpSimple->LoadSideInletEnth -
+                                               Psychrometrics::PsyHFnTdbW(simpleWatertoAirHP.OutletAirDBTemp,
+                                                                          simpleWatertoAirHP.OutletAirHumRat)); // Why doesn't this match QLoadTotal?
+        simpleWatertoAirHP.QSensible *= PartLoadRatio;
```

Comment:
Here is a subtle difference found when correcting this model. When using QLoadTotal/QSensible to set outlet conditions, those outlet conditions did not agree with QLoadTotal. i.e., QLoadTotal != Mdot * deltaH. For that reason, a reported QLoadTotalReport carries delivered capacity.